src/Controller/BoxFrontendController.php

```diff
@@ -14,9 +14,9 @@
 use Symfony\Component\Routing\Annotation\Route;
 
 /**
- * @Route("", name="frontend_")
+ * @Route("/box", name="box_frontend_")
```

Comment:
Are we changing the path for the machines to run against?

Reply:
That was the idea....

Reply:
Reason: A URL with a unique ID works best if it's "prefixed" by the entity to look up the id for. Else we risk url pattern clashes